Docs/Specifics/Vision.md

```diff
@@ -2,64 +2,113 @@
+
+### Pose Estimation
+We can also use AprilTags in a more sophisticated way to perform pose estimation.
+Once a tag and its corner locations have been detected, we can compare that to the known pose of that tag on the field and calculate an estimate of the distance, angle, etc. that the camera is from that tag.
+We know where the camera is relative to the rest of the robot, so we can then estimate where the robot is on the field.
+We'll also combine this estimate with the estimated pose returned by the wheels and gyro of the robot, which is known as odometry.
+This is useful because with a reliable pose estimation system, we can automate more of the scoring process.
+For example, in 2024 we were able to develop an accurate auto aim system based on the robot's estimated pose.
+
+<img src="../../Assets/Pose.PNG" alt="An example of pose estimation" width="600"/>
+
+Pose estimation is more accurate when there are multiple tags in frame.
+Each tag generates its own estimate, which can then be combined and used or discarded if one is inaccurate.
+
+How would an estimate be incorrect?
+One cause of this is pose ambiguity.
+
+![Demonstration of pose ambiguity](/Assets/Ambiguity.jfif)
+
+Humans are typically able to rely on things like depth perception and lighting to determine the difference between something like these two boxes, but it is more difficult for computers.
+If we also have pose estimates from other tags that are less ambiguous, we can throw out those (likely) incorrect estimates if they're significantly off.
+Having multiple cameras also helps with this, as another camera at a different angle could generate a more accurate pose with the same tag.
+Comparing a vision pose against odometry can also rule out incorrect poses.
+
+An important step to take before using 3D pose estimation is to correctly calibrate the camera.
+All camera lenses have some level of distortion that we need to account for when processing images gathered from them, especially in applications where it's important to accurately calculate where things are in 3D space.
+
+![Example of distorted chessboard](/Assets/Distortion.jpg)
+
+It's possible for us to undistort these images—we just need to find the camera-specific numbers that will allow us to perform that math, which we can do by calibrating the camera.
+There are a couple ways to do this, but one way is to take a lot of photos with the camera being calibrated of a chessboard pattern (which we know has straight lines and certain measurements) at different angles and positions and send those photos to programs that then handle the math for us and return those numbers.
+
+### Object Detection
+
+Sometimes we want to be able to detect other things on the field that aren't AprilTags.
+This could include game pieces, other robots, and more.
```

Comment:
change to "This primarily includes game pieces, but can include other robots or more"